test/parallel/test-fs-watchfile.js

```diff
@@ -19,7 +18,7 @@ assert.throws(function() {
   fs.watchFile(new Object(), function() {});
 }, /Path must be a string/);
 
-const enoentFile = path.join(fixtures, 'non-existent-file');
+const enoentFile = path.join(common.tmpDir, 'non-existent-file');
```

Comment:
Side note: If I were writing that `common.refreshTmpDir()` function again, I might name the function something like `refreshDirectory()` and have it take an argument so it was always invoked as `refreshDirectory(common.tmpDir)`. That might make it more clear what is going on there.